lamindb/examples/cellxgene/_cellxgene.py

```diff
@@ -143,6 +169,11 @@ def get_cxg_schema(
 
     from lamindb.models import Feature, Schema, ULabel
 
+    # Attempt to find the Schema early as building the Schema is expensive when looped
+    full_cxg_schema_name = f"AnnData of CELLxGENE version {schema_version} for {organism} of {', '.join(field_types) if isinstance(field_types, list) else field_types}"
```

Comment:
I'm not sure it's the best idea to rely on name matching here, I think relying on the hash would be better. But I get it takes a bit more time (which isn't terrible) and gives some warnings.